ui/pages/onboarding-flow/privacy-settings/index.scss

```diff
@@ -47,11 +47,6 @@
 
   &__network {
     position: relative;
-
-    .dropdown-menu-item {
-      font-size: 14px !important;
-      padding: 8px !important;
-    }
```

Comment:
Not used 

![Screenshot 2025-02-07 at 1 35 35 PM](https://github.com/user-attachments/assets/d65b0b6d-a3fa-476f-872d-ee34000f0ab4)